src/common/utility_unix.cpp

```diff
@@ -104,10 +112,12 @@ void setLaunchOnStartup_private(const QString &appName, const QString &guiName,
     }
 }
 
-static inline bool hasDarkSystray_private()
+#ifndef TOKEN_AUTH_ONLY
```

Comment:
I think you should remove this
if `TOKEN_AUTH_ONLY` is defined, the build would most probably fail ?